drivers/soundwire/bus.c

```diff
@@ -1273,6 +1286,11 @@ static int sdw_initialize_slave(struct sdw_slave *slave)
 	    !(slave->prop.quirks & SDW_SLAVE_QUIRKS_INVALID_INITIAL_PARITY)) {
 		/* Clear parity interrupt before enabling interrupt mask */
 		status = sdw_read_no_pm(slave, SDW_SCP_INT1);
+		if (status < 0) {
+			dev_err(&slave->dev,
+				"SDW_SCP_INT1 read failed:%d\n", status);
```

Comment:
now we have two identical error messages at two different code locations... I find this confusing when debugging stuff, so, maybe better make them different?

Reply:
added unique messages in follow-up commit.